swan-lake/integration-tutorials/transforming-message-formats.md

```diff
@@ -101,48 +100,125 @@ Follow the instructions given in this section to develop the service.
     ```
     $ bal new transforming-message-formats
     ```
+2. Remove the generated content in the `main.bal` file and open the diagram view in VS Code.
 
-2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file).
+    ![Open diagram view](/learn/images/integration-tutorials/transforming-message-formats/open_diagram_view.gif)
 
-    Import the
-    - `ballerina/http` module to develop the REST API and define the client that can be used to send requests to the backend services
-    - `ballerina/log` module to log debug, error, or info level information for each client request
+3. Generate record types corresponding to the payloads from the hospital service and input/output of the transforming function by providing samples of the expected JSON payloads.
```

Comment:
We haven't spoken about a "transforming function" before we get here, so may not make sense to the user.

I would have 

1. if there is a request payload, show generating the record for that. if not, show generating a record for the payload from the backend.
2. if there are other request or response payloads, say "Similarly, generate records ...". Don't have to give the JSON payloads here.
3. then say "The generated record types will be as follows." for all.

So basically, show one JSON -> record generation for whatever we encounter first. And then say generate similarly for others and show the records.